picli/styler/base.py

```diff
@@ -101,14 +118,20 @@ def zip_files(self, destination):
             f'{destination}/{self.name}.zip', 'w', zipfile.ZIP_DEFLATED
         )
         for file in self.run_config.files:
-            if file['styler'] == f'{self.name}':
-                if self.config.debug:
-                    message = f'Writing {file["file"]} to zip'
-                    LOG.info(message)
-                zip_file.write(
-                    f"{self.config.base_config.base_dir}/{file['file']}",
-                    file['file']
-                )
+            if self.pipe_config.debug:
+                message = f'Writing {file["file"]} to zip'
+                LOG.info(message)
+            zip_file.write(
+                file['file'],
+                os.path.relpath(file['file'], self.pipe_config.base_config.base_path))

```

Comment:
util.good_path or some such

Reply:
I think I'll save that for a separate PR.